src/Components/Homepage/Homepage.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import { Button } from "react-bootstrap";
+import styles from "./Homepage.module.css";
+
+type Props = {
+    openModal: () => void;
+}
+
+export const Homepage: React.FC<Props> = ({
+    openModal,
+}) => {
+
+    return (
+    <div className={styles.CanvaHomePage}>
+        <Button
+        variant="info"
+        onClick={openModal}
+        className={styles.createDesignBtn}
+      >
+        Create a design
+      </Button>
+    </div>
+    )
+}
```

Comment:
nit: new line at end of each file.

Reply:
I have removed the extra lines, but there isn't an extra line on this file? Should I move the } bracket up?

Reply:
At canva, we prefer to always have a new line at the end of each file.